zboss/CHANGELOG.rst

```diff
@@ -9,6 +9,22 @@ Changelog
 
 All notable changes to this project in the |NCS| are documented in this file.
 
+Main branch
+***********
+
+All the notable changes included in the |NCS| main branch are documented in this section.
+
+Changes
+=======
+
+* Added APS commands dups detection.
+* Fixed ZR going out of capacity due to many joins/rejoins.
+* Added NCP protocol version checks.
+* Fixed unsolicited NCP commands not being handled correctly.
+* Removed precompiled libraries for Zigbee Green Power Combo Basic functionality.
+* Updated the ZBOSS stack to version ``3.9.0.0+4.0.1``.
```

Comment:
`3.9.0.1` due to documentation build issues?

Reply:
Will be changed - waiting for new sdk to be built

Reply:
Changed to 3.9.0.1, marking as resolved